### ISSUE TITLE
bootstrap fetch sizes are ignored by mysql connector

### ISSUE THREAD

**Issue by mcleae** [zendesk/maxwell#335]
Was attempting to bootstrap a large table and no matter what I set the fetch size to the results failed to stream in.  Looking at the mysql connector documentation [here](https://dev.mysql.com/doc/connector-j/5.1/en/connector-j-reference-implementation-notes.html) (specifically the result set section) I found that in order to get row by row streaming the statement needs to be configured as follows
```
stmt = conn.createStatement(java.sql.ResultSet.TYPE_FORWARD_ONLY,
              java.sql.ResultSet.CONCUR_READ_ONLY);
stmt.setFetchSize(Integer.MIN_VALUE);
```

Trying this however with connections obtained from the snaq.db.ConnectionPool (used by the replication connection in maxwell) I got "No database selected" errors when any query was executed.  I worked around that by instead using the DriverManager to get a connection as follows
```
protected Connection getStreamingConnection() throws SQLException {
        Connection conn = DriverManager.getConnection(context.getConfig().replicationMysql.getConnectionURI(), context.getConfig().replicationMysql.user, context.getConfig().replicationMysql.password);
        conn.setCatalog(context.getConfig().databaseName);
        return conn;
    }
```

With the above connection and the batch statement configured as per the documentation I was able to bootstrap a 42mm row 5gb table.
```
try ( Connection connection = getConnection();
              Connection streamingConnection = getStreamingConnection()) {
			setBootstrapRowToStarted(startBootstrapRow, connection);
			ResultSet resultSet = getAllRows(databaseName, tableName, schema, streamingConnection);
```

Happy to submit a PR if you agree that this is an issue

**Comment by mcleae** [comment]
btw, according to the connector documentation the connection used to retrieve the streamed result set shouldn't be used to issue any more queries until the entire result has been fetched so two connections are needed since you update the bootstrap table as rows are processed.

**Comment by osheroff** [comment]
yes, PR plz

**Comment by mcleae** [comment]
pr https://github.com/zendesk/maxwell/pull/336

**Comment by osheroff** [comment]
fixed in 1.1.0-pre4
